assembl/graphql/idea.py

```diff
@@ -108,7 +126,13 @@ def resolve_total_sentiments(self, args, context, info):
         return self.get_total_sentiments()
 
 
-class IdeaAnnoucement(SecureObjectType, SQLAlchemyObjectType):
+class IdeaAnnouncementInput(graphene.InputObjectType):
+    __doc__ = docs.IdeaAnnouncement.__doc__
+    title_entries = graphene.List(LangStringEntryInput, description=docs.IdeaAnnouncement.title_entries)
+    body_entries = graphene.List(LangStringEntryInput, description=docs.IdeaAnnouncement.body_entries)
+
```

Comment:
no required field here?

Reply:
You're right, should be required, to have better flow definition to not check for null.